packages/util/.eslintrc.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   extends: ["@pcd/eslint-config-custom"],
   root: true,
+  rules: {
+    "@typescript-eslint/no-non-null-assertion": "off"
+  }
```

Comment:
I'm not sure we should do this

Reply:
there are a few ways of getting around this. you could
- make `initCryptoAPI` return `crypto`
- typescript has this [assertion function](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions) api 

strongly prefer refactoring code to pass the lints instead of changing the lints

Reply:
I've tried to make the function return `crypto` but I realized it would re-assign that value every time the function is called. I'm not sure how the dynamic `require` works and I didn't want to execute that code every time.

The assertion function is a nice feature, I didn't know it! However, the `initCryptoAPI` would need a parameter to make `asserts crypto` work.

What would you prefer?

Reply:
I added two suggestion comments

Reply:
Great, thank you! Ready to be merged